### PR TITLE
Fix the simple_make example so that it works on Mac & Linux

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

**Comment by nomis52** [comment]
@googlebot I signed it!

**Comment by nomis52** [comment]
@googlebot I signed it!

File: examples/simple_make/code/Makefile [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 default all $(BUILD_DIR)/lib/liba.a: liba.cpp liba.h
 	rm -rf $(BUILD_DIR)/lib
 	mkdir -p $(BUILD_DIR)/lib
 	clang -fPIC -o $(BUILD_DIR)/lib/liba.o -c liba.cpp
 	ar rcs $(BUILD_DIR)/lib/liba.a $(BUILD_DIR)/lib/liba.o
 
 install: $(BUILD_DIR)/lib/liba.a
-	cp -rv $(BUILD_DIR)/lib --target-directory=$(PREFIX)
-	cp liba.h --target-directory=$(PREFIX)/include
+	ls $(PREFIX)
```
Comment (irengrig) [inline examples/simple_make/code/Makefile]: Please remove ls call, is that needed?

Comment (nomis52) [reply examples/simple_make/code/Makefile]: Done

**Comment by nomis52** [comment]
> Thank you for contributing!
> Do you also want to add this test to the set of Mac tests? (and probably rename from cmake_tests to mac_tests ?)
> https://github.com/bazelbuild/rules_foreign_cc/blob/adb04eed2c058b4b161b321b1104eb55ef63b0f4/.bazelci/config.yaml#L23

Ah nice, I didn't know about that. Done

**Pull request merged**